engines/python/setup/djl_python/rolling_batch/rolling_batch_vllm_utils.py

```diff
@@ -279,7 +280,9 @@ def get_multi_modal_data(request: Request) -> dict:
 
 
 def get_prompt_inputs(request: Request):
-    prompt_inputs: PromptInputs = {"prompt": request.request_input.input_text}
+    prompt_inputs: PromptInputs = {
+        "prompt": json.dumps(request.request_input.input_text)
```

Comment:
I don't think this fix the issue.

We don't support rolling batch with client side batch, We should fail the request instead of treat it as string.